addons/web/static/src/views/widgets/ribbon/ribbon.js

```diff
@@ -0,0 +1,51 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+import { standardViewWidgetProps } from "../standard_view_widget_props";
+
+const { Component } = owl;
+
+/**
+ * This widget adds a ribbon on the top right side of the form
+ *
+ *      - You can specify the text with the title attribute.
+ *      - You can specify the tooltip with the tooltip attribute.
+ *      - You can specify a background color for the ribbon with the bg_color attribute
```

Comment:
maybe talk about *props* instead of *attribute* (makes more sense from a Component point of view)